Description: Fix detection of <linux/errqueue.h>
 Linux ≥ 3.17 requires the definition of `struct timespec`.
Author: Peter Colberg <peter@colberg.org>
Forwarded: https://github.com/LubosD/twinkle/pull/42
Last-Update: 2015-12-30
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -23,6 +23,7 @@ if (WITH_QT4 AND WITH_QT5)
 endif (WITH_QT4 AND WITH_QT5)
 
 include (CheckIncludeFile)
+include (CheckIncludeFiles)
 
 find_package(LibXml2 REQUIRED)
 find_package(LibMagic REQUIRED)
@@ -130,7 +131,7 @@ endif (WITH_G729)
 
 check_include_file(unistd.h HAVE_UNISTD_H)
 check_include_file(linux/types.h HAVE_LINUX_TYPES_H)
-check_include_file(linux/errqueue.h HAVE_LINUX_ERRQUEUE_H)
+check_include_files("sys/socket.h;linux/errqueue.h" HAVE_LINUX_ERRQUEUE_H)
 
 set(datadir "${CMAKE_INSTALL_PREFIX}/share/twinkle")
 configure_file(twinkle_config.h.in twinkle_config.h)
